AspireWithSentry.ServiceDefaults/Extensions.cs

```diff
@@ -0,0 +1,146 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using Sentry.AspNetCore;
+using Sentry.OpenTelemetry;
+
+namespace Microsoft.Extensions.Hosting;
+
+public static class Extensions
+{
+    public static IHostApplicationBuilder AddServiceDefaults(this IHostApplicationBuilder builder)
+    {
+        builder.ConfigureOpenTelemetry();
+
+        builder.AddDefaultHealthChecks();
+
+        builder.Services.AddServiceDiscovery();
+
+        builder.Services.ConfigureHttpClientDefaults(http =>
+        {
+            // Turn on resilience by default
+            http.AddStandardResilienceHandler();
+
+            // Turn on service discovery by default
+            http.UseServiceDiscovery();
+        });
+
+        return builder;
+    }
+
+    public static WebApplicationBuilder AddSentry(this WebApplicationBuilder builder, Action<SentryAspNetCoreOptions>? configureOptions = null)
+    {
+        builder.WebHost.UseSentry(options =>
+        {
+            // Change this to the DSN of your Sentry project
+            // options.Dsn = "... your DSN here ...";  // Replace this!!!

```

Comment:
if we uncomment, and assign it to an invalid value, will it make it easier for folks to find out what to change?

`= "create a project in sentry.io to get this value";`

Reply:
I've added an explicit exception that will throw until the instructions from the README.md have been completed.

Reply:
That's a pretty cool solution!